.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: php
+dist: bionic
 php:
+    - 8.1.0
     - 8.0
     - 7.3
     - 7.2
     - 7.1
-    - 7.0
```

Comment:
Maybe I've fixed that compilation error in master branch :)
So would you consider save it for a while?

Reply:
Yeah I can revert that if you fixed it !

Reply:
Looks like the pho 7.0 build is still failing